### PR TITLE
Update dataset.proto

### PR DESCRIPTION
**Pull request by xmyqsh**
fix the comment of rotation calculation from the angular velocity

### CODE REVIEW

**Comment by xmyqsh** [comment]
@googlebot I signed it!

File: waymo_open_dataset/dataset.proto [hunk 1]
```diff
@@ -231,7 +231,7 @@ message CameraImage {
   // at *global* frame.
   // With this velocity, the pose can be extrapolated.
   // r(t+dt) = r(t) + dr/dt * dt where dr/dt = v_{x,y,z}.
-  // R(t+dt) = R(t) + R(t)*SkewSymmetric(w_{x,y,z})*dt
+  // R(t+dt) = R(t)*SkewSymmetric(w_{x,y,z})*dt
```
Comment (peisun1115) [inline waymo_open_dataset/dataset.proto]: This is not accurate either based on my understanding. I think the most accurate form is

R(t+dt) = exp(Wt)R(t) where W is the SkewSymetric form of the angular velocity. I will update the comment

Comment (xmyqsh) [reply waymo_open_dataset/dataset.proto]: @peisun1115 You are right.
https://github.com/waymo-research/waymo-open-dataset/pull/81

Comment (xmyqsh) [reply waymo_open_dataset/dataset.proto]: @peisun1115
这里用反对称方程也不对

**Comment by xmyqsh** [comment]
@peisun1115 
I have created a new PR #82 to solve all related problems. Please have a double check on the intrinsic rotation definition [https://github.com/waymo-research/waymo-open-dataset/blob/e6cf03bb5925579a44efac830e864fb8660b5d2a/waymo_open_dataset/dataset.proto#L212-L216](https://github.com/waymo-research/waymo-open-dataset/blob/e6cf03bb5925579a44efac830e864fb8660b5d2a/waymo_open_dataset/dataset.proto#L212-L216)

File: waymo_open_dataset/dataset.proto (hunk 1 above)
Comment (peisun1115) [reply waymo_open_dataset/dataset.proto]: dR/dt = W*R
This can be solved to:
R = exp(W) * R(0)

When dt is small, R(t+dt) = R(t) + WR(t)

**Pull request merged**